### PR TITLE
[7619] image deletion docs

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [x] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [x] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: apps/debate/views.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def get_permission_object(self):
 class SubjectDeleteView(
     mixins.DashboardBaseMixin,
     mixins.DashboardComponentMixin,
-    mixins.DashboardComponentDeleteSignalMixin,
+    mixins.DashboardComponentFormSignalMixin,
```
Comment (m4ra) [inline apps/debate/views.py:99]: @goapunk this has been deprecated for a while. and we had left a trial period when we switched to `form_valid` for deletion.

**Comment by m4ra** [comment]
@goapunk will add few more tests for the modules in a separate commit, but you can start reviewing it.

File: docs/deletion.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Deletion of images conforming to DSGVO
```
Comment (goapunk) [inline docs/deletion.md:1]: For all of these cases: should we add a sentence explaining that an old image gets deleted when it's a replaced by a new one (and not just when e.g. an idea is deleted) ?

**Review by goapunk — COMMENTED** [review]
Small comment about image replacement and should we also add tests which checks that the old image is deleted when uploading a new one? (can be in a separate PR)

File: docs/deletion.md (hunk 2 above)
Comment (goapunk) [reply docs/deletion.md:1]: also one of the tests is failing :)

**Comment by m4ra** [comment]
@goapunk there is a test for users replacing images, and I added for project/organisation and interactiveevent. i won't add for the other modules as should be same behaviour as the interactiveevent.

**Pull request merged**